src/tmpfiles/tmpfiles.c

```diff
@@ -110,6 +110,14 @@ typedef enum ItemType {
         ADJUST_MODE = 'm', /* legacy, 'z' is identical to this */
 } ItemType;
 
+typedef enum AgeBy {
+        AGE_BY_UNSET = 0,
+        AGE_BY_ATIME = 1 << 0,
+        AGE_BY_BTIME = 1 << 1,
+        AGE_BY_CTIME = 1 << 2,
+        AGE_BY_MTIME = 1 << 3,
+} AgeBy;
```

Comment:
hmm, why have AGE_BY_UNSET here?

i.e. i think our default behaviour should be something we encode here as AGE_BY_DEFAULT or so, which is just a combo of bits we define here.

Reply:
Well, `AGE_BY_UNSET` reverts back to the default behavior in `dir_cleanup()`, where the loop `continue`s as soon "too new" entry is found immediately instead of checking for the rest of the file timestamp types, and then checking for the masks. Isn't that faster?

Reply:
I think I understand what you meant, dropped `AGE_BY_UNSET` for `AGE_BY_DEFAULT`.